backend/src/appointment/routes/caldav.py

```diff
@@ -37,6 +37,13 @@ def caldav_autodiscover_auth(
 
     dns_lookup_cache_key = f'dns:{utils.encrypt(connection.url)}'
 
+    # Check for an attempt to use Google CalDAV API
+    # which we don't support because we use their API directly
+    basename = urlparse(connection.url).netloc
+    google_domains = ['googleusercontent.com', 'google.com', 'gmail.com']
```

Comment:
Should these be put in `backend/src/appointment/defines.py` so they're easier to update in the future?

Reply:
Yes, of course! Thanks.